movieReviewApp/src/main/java/com/mobiquity/movieReviewApp/controller/UserController.java

```diff
@@ -32,35 +33,42 @@ public UserController(SignUpService signUpService, PasswordRecoverService passwo
   }
 
   @PostMapping("/signUp")
-  public String signUp(@RequestBody UserProfile userProfile) {
-    return signUpService.saveUser(userProfile);
+  public ResponseEntity<Object> signUp(@RequestBody UserInformation userInformation) {
```

Comment:
to add status code so that front enders will know how to style their return message (httpstatus code 200 or unauthorised).ofCourse we can add field in responseMovieapp I prefer responseEntity way(just in case if there is need for header).